ultralytics/yolo/data/base.py

```diff
@@ -30,6 +31,7 @@ def __init__(
         imgsz=640,
         cache=False,
         augment=True,
+        transforms=None,
```

Comment:
Adding a new parameter in the middle here of a non-kwarg-only parameter list will break usage for people who are currently passing parameters by position.

It would be better if this was a kwarg-only constructor, but maybe `transforms` could be the first kwarg-only argument.

Reply:
Same as above.  Thanks!

Reply:
See above :)

Reply:
Same